index.bs

```diff
@@ -1241,16 +1261,22 @@ Create a constant {{MLOperand}} that can be used in {{MLGraphBuilder}} methods.
         The permissions and context validity have been checked by [[#api-mlgraphbuilder-constructor]] steps.
     </div>
     1. Let |value| be the first argument.
-        1. If |value| is not a number, then  throw a "{{TypeError}}" {{DOMException}} and stop.
+        1. If |value| is an array, then let |shape| be the result of invoking [=MLOperand/shape=] given |value|.
```

Comment:
Should it just throw error if `|value|` is not a number, according to the current [IDL definition](https://www.w3.org/TR/webnn/#dom-mlgraphbuilder-constant-value-type) of `MLGraphBuilder.constant(value, type)`? If this is a new proposal, it probably should be in a separate issue.

Reply:
I thought a constant could be a scalar or a tensor? So here it's just checking if it's an array, and infers the dimensions/shape of the array as a basis to create the descriptor for the new MLOperand.

Reply:
> I thought a constant could be a scalar or a tensor?

Yes. [`constant(value, type)`](https://www.w3.org/TR/webnn/#dom-mlgraphbuilder-constant-value-type) method builds a scalar constant. [`constant(descriptor, bufferView)`](https://www.w3.org/TR/webnn/#dom-mlgraphbuilder-constant) method builds a tensor (`descriptor.dimensions` is present) or scalar constant (`descriptor.dimensions` is not present) .

> So here it's just checking if it's an array, and infers the dimensions/shape of the array as a basis to create the descriptor for the new MLOperand.

The current IDL [defines](https://www.w3.org/TR/webnn/#dom-mlgraphbuilder-constant-value-type) the `value` parameter as a double. If it is an array, it should report error.